### PR TITLE
adding 2 folders

### PR DESCRIPTION
**Pull request by sunayana47**

### CODE REVIEW

File: basics/ex1.py [hunk 1]
```diff
@@ -1 +1,2 @@
-print("Hello")
```
Comment (sriram9494) [inline basics/ex1.py:1]: why did you delete the code??

Comment (sunayana47) [reply basics/ex1.py:1]: this is no longer required
